salt/modules/pillar.py

```diff
@@ -328,7 +328,7 @@ def obfuscate(*args, **kwargs):
 
 # naming chosen for consistency with grains.ls, although it breaks the short
 # identifier rule.
-def ls(*args):
+def ls(*args, **kwargs):
```

Comment:
I hate blindly accepting keyword arguments....

Could we be more explicit on what this function really accepts?

Reply:
When I say, I hate, it's because I see problems when Salt eventually implements function call validation, data casting, etc....

Using `**kwargs` will prevent that, or at least bypass validation on those keyword arguments...

Reply:
Agreed, but this function is just passing off to `items()`... which accepts `*args` and `**kwargs`. I'd rather not fundamentally change a function unrelated to my issue in this PR, which is what would be required to fulfill this request.

Reply:
You're not changing an unrelated function, you'd only be changing the problematic function